File: Python/1013 - The Greatest.py
a, b, c = map(int, input().split())

maiorab = (a + b + abs(a - b))/2
maiorac = (maiorab + c + abs(maiorab - c))/2

print(f"{int(maiorac)} eh o maior")

# One-liner
# print(f"{int(((a + b + abs(a - b))/2 + c + abs((a + b + abs(a - b))/2 - c))/2)} eh o maior")

File: C/1013 - The Greatest.c
#include <stdio.h>
#include <stdlib.h>
 
int main() {
    int a, b,c, maiorab, maiorac;
    
    scanf("%d %d %d", &a, &b, &c);
    
    maiorab = (a + b + abs(a-b))/2;
    maiorac = (maiorab + c + abs (maiorab -c))/2;
    
    printf("%d eh o maior\n", maiorac);

    return 0;
}
